### PR TITLE
feat: integrate infrastructure

### PR DESCRIPTION
**Pull request by wannacfuture**
Resolves #

Quality Assurance:

### CODE REVIEW

File: src/handlers/comment/handlers/issue/issue-closed.ts [hunk 1]
```diff
@@ -23,23 +23,46 @@ export async function issueClosed(context: Context) {
...
 
-  // 1. score sources will credit every contributor for every one of their contributions
-  const sourceScores = await aggregateAndScoreContributions({
-    context,
-    issue,
-    issueComments,
+  const pullRequestComments = await getPullRequestComments(context, owner, repository, issueNumber);
+  const repoCollaborators = await getCollaboratorsForRepo(context);
+
+  await dispatchWorkflow("wannacfuture", "UbiquitBot-Bridge", "bridge.yml", {
```
Comment (wannacfuture) [inline src/handlers/comment/handlers/issue/issue-closed.ts:29]: The only thing we need to consider is this line.

Should we set the path as the same organization, and `ubiquibot-config` repo?

And the user can set the config repo as my bridge repo.
@pavlovcik

**Review by wannacfuture — COMMENTED** [review]
@pavlovcik

File: src/handlers/comment/handlers/issue/issue-closed.ts (hunk 1 above)
Comment (wannacfuture) [reply src/handlers/comment/handlers/issue/issue-closed.ts]: Fixed.

**Review by 0x4007 — CHANGES_REQUESTED** [review]
Do me a favor. There's several `any` type casts. I would appreciate if you spend another hour or so setting specific types for type safety wherever you can find, especially in the new code you introduced. Good work on the pull request!

**Comment by pavlovcik** [comment]
This is an interesting approach. For some reason I figured that the `secretToken` would be set in the `ubiquibot-config` repo secrets instead. Is transmitting it from the other repo the superior choice? If so, why?

https://github.com/wannacfuture/ubiquibot-config/blob/d3e49c8f44183a033459cb1257695df4018c95a5/.github/workflows/bridge.yml#L21-L23

**Comment by wannacfuture** [comment]
> This is an interesting approach. For some reason I figured that the `secretToken` would be set in the `ubiquibot-config` repo secrets instead. Is transmitting it from the other repo the superior choice? If so, why?
> 
> https://github.com/wannacfuture/ubiquibot-config/blob/d3e49c8f44183a033459cb1257695df4018c95a5/.github/workflows/bridge.yml#L21-L23

We need to return the response to the original repo.
And for that, we need to put secret token as the payload for the request.
https://github.com/wannacfuture/ubiquibot-config/blob/d3e49c8f44183a033459cb1257695df4018c95a5/.github/workflows/bridge.yml#L42C1-L48C172

**Comment by pavlovcik** [comment]
I think it makes sense to rename `bridge.yml` to `compute.yml` or something of the like, because I don't want it to be confused with bridges as we see in DeFi.

**Comment by pavlovcik** [comment]
Is there any extra setup required other than pushing `bridge.yml` to `@pavlovcik/ubiquibot-config`? I'm leaning towards merging this so I can continue working.

**Pull request merged**